go.mod

```diff
@@ -91,6 +91,7 @@ require (
 	google.golang.org/grpc v1.29.1
 	google.golang.org/protobuf v1.25.0
 	gopkg.in/freddierice/go-losetup.v1 v1.0.0-20170407175016-fc9adea44124
+	gopkg.in/fsnotify.v1 v1.4.7
```

Comment:
this should be github.com/fsnotify/fsnotify (see PR from @AlekSi  on that), we already have this import